packages/modal/src/ModalPanel.tsx

```diff
@@ -58,10 +56,8 @@ export const ModalPanel = ({
 				style={animationStyles}
 			>
 				<ModalTitle id={titleId}>{title}</ModalTitle>
-				<Flex gap={2} flexGrow={1} flexDirection="column">
-					{children}
-					{actions}
-				</Flex>
+				<Box>{children}</Box>
+				{actions ? <Box paddingTop={1}>{actions}</Box> : null}
```

Comment:
```suggestion
				{actions}
```

Can we put this padding on the `ModalButtonGroup`?